src/wallet/rpcwallet.cpp

```diff
@@ -3162,6 +3232,7 @@ static const CRPCCommand commands[] =
 { //  category              name                        actor (function)           okSafeMode
     //  --------------------- ------------------------    -----------------------    ----------
     { "rawtransactions",    "fundrawtransaction",       &fundrawtransaction,       false,  {"hexstring","options"} },
+    { "wallet",             "signrawtransactionwithwallet",       &signrawtransactionwithwallet,       false, {"hexstring","prevtxs","sighashtype"} },
```

Comment:
alignment

Reply:
Aligning this is hard

Reply:
why? Can be done in a whitespace only commit at the end if you're worried about the diff

Reply:
Done